README.md

```diff
@@ -310,6 +311,10 @@ object-inspect@1.0.2
 stevemao/left-pad
 ```
 
+### Copy system certificate preferences when installing
+
+If you have any of `ca`, `cafile`, or `cert` present in your npm config when installing a new version, those keys will be copied into the newly-installed version's global config file. If you don't want this behavior, pass the `--skip-system-certs` flag when installing a new version
```

Comment:
Can you explain more about why this complexity is desired or needed?

I don't think having these present is a common case, and looking these up from the current node version (which might be "the system node") will slow things down for the majority that would never need it.

It seems simpler for the user to manually rerun their npm config commands, or better, for each project to define its own `.npmrc` file.